### DISCUSSION TITLE
Blog Support

### DISCUSSION THREAD

**Discussion by anejjar** [shopperlabs/framework#157] [Ideas]
for every ecommerce website it's nice to have a blog section so it would be good to add a way to create articles and manage seo and comments later

**Comment by nextgenthing** [comment]
I can't see why you can't create or use an existing Laravel package/repo just deals with blogs. This is a eCommerce framework I don't think it should do more than that.

**Comment by bdsumon4u** [comment]
A blog is a totally different thing. But the promised features (`Soon`) should be completed **soon**.
![image](https://github.com/shopperlabs/framework/assets/75123992/d7ab6f3e-ac21-4bda-acb9-4476cd1a21c9)

**Comment by mckenziearts** [comment]
Hey @anejjar, agreeing with @nextgenthing here: a blog sits outside the scope of a headless e-commerce admin, so we're keeping it out of core on purpose.

That said, it fits very well as a community addon, and Laravel already has excellent options you can drop next to Shopper (Wink, Canvas, Filament Curator, or a simple custom module). If someone from the community wants to publish a `shopper-blog` addon, we'll happily feature it in the Addons section of the docs.
